stories/core/Table.stories.tsx

```diff
@@ -2500,3 +2503,139 @@ DraggableColumns.args = {
   enableColumnReordering: true,
   isSelectable: true,
 };
+
+export const ColumnManager: Story<Partial<TableProps>> = (args) => {
+  type TableStoryDataType = {
+    index: number;
+    name: string;
+    description: string;
+    id: string;
+    startDate: Date;
+    endDate: Date;
+  };
+  const columns = useMemo(
+    (): Column<TableStoryDataType>[] => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'index',
+            Header: '#',
+            accessor: 'index',
+            disableToggleVisibility: true,
+          },
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            fieldType: 'text',
+          },
+          {
+            id: 'id',
+            Header: 'ID',
+            accessor: 'id',
+          },
+          {
+            id: 'startDate',
+            Header: 'Start date',
+            accessor: 'startDate',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.startDate.toLocaleDateString('en-US');
+            },
+          },
+          {
+            id: 'endDate',
+            Header: 'End date',
+            accessor: 'endDate',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.endDate.toLocaleDateString('en-US');
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+  const data = useMemo(
```

Comment:
You need to also add `data` to `args` or just disable changing `data` trough `Controls`.

Reply:
Added `data` to `args`